appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -292,18 +292,23 @@ public MockForm(SimpleEditor editor) {
     setScrollableProperty(getPropertyValue(PROPERTY_NAME_SCROLLABLE));
   }
 
-  public void changePreviewSize(boolean isTablet) {
-    if (isTablet) {
-      PORTRAIT_WIDTH   = TABLET_PORTRAIT_WIDTH;
-      PORTRAIT_HEIGHT  = TABLET_PORTRAIT_HEIGHT;
-      LANDSCAPE_WIDTH  = TABLET_LANDSCAPE_WIDTH;
+  public void changePreviewSize(int width, int height) {
+    // It will definitely be modified in the future to add more options.
+    if (width == TABLET_PORTRAIT_WIDTH) {
```

Comment:
Are these extra branches necessary at this point? If you're always passing in width and height then whichever if path is taken should reduce to the else case anyway (at least that's how it seems to me). If so, then you might be able to move the constants PHONE_* and TABLET_* into SimpleVisibleComponentsPanel to use in place of the numeric values there.

Reply:
Done